en/components/chip.md

```diff
@@ -121,21 +121,6 @@ public chipRemoved(event: IBaseChipEventArgs) {
     this.changeDetectionRef.detectChanges();
 }
 ```
-
-### Dragging
```

Comment:
I don't see duplicate section of this one. We'll lose this content if removed.

Reply:
The chip draggable property is also mentioned in the Reorder Chips section

Reply:
Added @skrustev for a review